### PR TITLE
[Website]fix the method params output in doc

### PR DESCRIPTION
**Pull request by sunnylqm**
if `this.props.params.length` is 0, then the result will be 0
screen shot 2016-11-08 at 20 51 07

### CODE REVIEW

**Comment by sunnylqm** [comment]
@hramos

**Comment by hramos** [comment]
Is the current behavior that `0` will be rendered when a method has 0 params? Can you post an "after" screenshot of what the docs look like after the fix is applied?

**Comment by sunnylqm** [comment]
@hramos  I tried to catch an "after" screenshop, but it took hours for me to clone this repo and npm install...so I cancelled.
But I can give you this:
the original code is 
```
this.props.params && this.props.params.length && this.props.params.map(...)
```
Let's mimic the result:
qq 20161109153336

**Comment by sunnylqm** [comment]
@hramos ?

**Comment by sunnylqm** [comment]
@janicduplessis ?

**Comment by mkonicek** [comment]
What does this change do? Would it render `getString()` in the docs, rather than `getString(0)`?

**Comment by mkonicek** [comment]
I can verify this fixes the bug. Here's an after screenshot:

screenshot 2016-11-23 16 04 20

Thanks for finding it! This was very clowny and we've had it in the docs for a long time.

I edited the code a bit to make it clearer we're checking for a non-empty array.

@facebook-github-bot shipit

File: website/layout/AutodocsLayout.js [hunk 1]
```diff
@@ -654,7 +654,7 @@ var Method = React.createClass({
           </span> || ''}
           {this.props.name}
           <span className="methodType">
-            ({this.props.params && this.props.params.length && this.props.params
+            ({this.props.params && (this.props.params.length !== 0) && this.props.params
```
Comment (sunnylqm) [inline website/layout/AutodocsLayout.js]: @mkonicek 
The point here I guess is to check whether `params` is an Array (by checking whether length exists).
If `params` is 100% array, then this length check can be simplily removed. If length is 0, map function will return an empty array, which has no problem in JSX.

Comment (sunnylqm) [inline website/layout/AutodocsLayout.js]: But if `params` can be types other than array, then `this.props.params.length !== 0` is unsafe.

**Comment by sunnylqm** [comment]
@mkonicek The point here I guess is to check whether params is an Array (by checking whether length exists).
If params is 100% array, then this length check can be simplily removed. If length is 0, map function will return an empty array, which has no problem in JSX.
But if params can be types other than array, then this.props.params.length !== 0 is unsafe.

**Comment by hramos** [comment]
@mkonicek do you still want to import this? Looks like that was the intention, but the GitHub bot, I believe, will only handle the shipit command when it's a standalone comment.

Both tests aren't passing as of this writing, but I've restarted the builds to verify.

**Comment by lacker** [comment]
It looks like this is failing in Circle and the failure is legitimate.

**Comment by hramos** [comment]
Let's see if tests are still failing.

**Pull request closed without merging**